### PR TITLE
DEVCVL-273 - [FE] Create/Edit Collection UI-UX improvements

### PR DESCRIPTION
**Pull request by pracheebapate**

### CODE REVIEW

**Review by kyawthetkt — APPROVED** [review]

File: src/pages/collections/single/CreateEdit.js [hunk 1]
```diff
@@ -854,18 +874,56 @@ const CreateEditCollection = () => {
...
+    const handleCreateCollection = () => {
+        setShowSuccessModel(false);
+        setFormStatus('');
+        form.resetFields();
+        window.location = '/collection/create';
+    }
+
+    const handleShowMyCollections = () => {
+        setShowSuccessModel(false);
+        navigate('/my-collections');
```
Comment (shakil-adv) [inline src/pages/collections/single/CreateEdit.js:924]: Please navigate to `user/userAddress/collections`

**Review by shakil-adv — CHANGES_REQUESTED** [review]

File: src/pages/collections/single/CreateEdit.js [hunk 2]
```diff
@@ -777,31 +803,27 @@ const CreateEditCollection = () => {
...
-                    content:'Collection saved successfully!',
-                    style: { marginTop: '20vh'}
-                });
-
-                // Clear form data and reset image URLs
-                // form.resetFields();
-
                 // Enable Publish Collection button
-                if(collection.get('contractAddress') === undefined) {
+                if(collection.get('contractAddress') === undefined
```
Comment (jibrail-idris) [inline src/pages/collections/single/CreateEdit.js:815]: Javascript has two possible not defined values
`undefined` or `null`.

File: src/pages/collections/single/CreateEdit.js (hunk 1 above)
Comment (pracheebapate) [reply src/pages/collections/single/CreateEdit.js]: Done.

File: src/pages/collections/single/CreateEdit.js (hunk 2 above)
Comment (pracheebapate) [reply src/pages/collections/single/CreateEdit.js]: Done changed to `if(!collection.get('contractAddress'))`

**Review by shakil-adv — APPROVED** [review]

**Pull request merged**